src/input/api/Wiimote/windows/WinWiimoteDevice.cpp

```diff
@@ -3,6 +3,9 @@
 #include <hidsdi.h>
 #include <SetupAPI.h>
 
+#pragma comment(lib, "Setupapi.lib")
+#pragma comment(lib, "hid.lib")
```

Comment:
Otherwise the linker complains about undefined symbols, like for example `HidD_GetAttributes`.
Weirdly enough if I put those `#pragma` statements back in `nsyshid.cpp` everything links fine, but having them here seems like a cleaner solution.